.eslintrc

```diff
@@ -1,15 +1,33 @@
 {
+  "ecmaFeatures": {
+    "jsx": true,
+    "modules": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "parser": "babel-eslint",
   "extends": "eslint-config-airbnb",
   "rules": {
+    "no-loop-func": 0,
+    "quotes": [2, "single"],
+    "strict": [2, "never"],
+    "react/jsx-uses-react": 2,
```

Comment:
These are redundant as they're already set in the airbnb eslint config https://github.com/airbnb/javascript/blob/fbf81eba3a02ef2dc4c92b0d98f93a4bad99869d/packages/eslint-config-airbnb/rules/react.js#L70

Only add rules here if we need to override eslint defaults or rules we inherit from airbnb.